misk/src/test/kotlin/misk/client/ClientMetricsInterceptorTest.kt

```diff
@@ -63,17 +66,18 @@ internal class ClientMetricsInterceptorTest {
 
   @Test
   fun responseCodes() {
-    assertThat(requestDuration.labels("pinger.ping", "all").get().count).isEqualTo(6)
-    assertThat(requestDuration.labels("pinger.ping", "202").get().count).isEqualTo(1)
-    assertThat(requestDuration.labels("pinger.ping", "4xx").get().count).isEqualTo(2)
-    assertThat(requestDuration.labels("pinger.ping", "404").get().count).isEqualTo(1)
-    assertThat(requestDuration.labels("pinger.ping", "403").get().count).isEqualTo(1)
-    assertThat(requestDuration.labels("pinger.ping", "5xx").get().count).isEqualTo(1)
-    assertThat(requestDuration.labels("pinger.ping", "503").get().count).isEqualTo(1)
+    assertThat((requestDuration.record("pinger.ping", "all") as io.prometheus.client.Histogram.Child).get().count).isEqualTo(6)
```

Comment:
you shouldn't need to cast this. just add a `count` method to the base `Histogram` interface. I think that's a reasonable method to add